File: LOO 2°Parcial/Java/Vehiculo.java
public abstract class Vehiculo {
  public String matricula;
  public float velocidad;
  public float cantidad;

  // me pasan la matricula como parametro y la velocidad no es necesaria ya que es 0
  Vehiculo(String matricula) {
    this.matricula = matricula;
    this.velocidad = 0;
  }

  // declaro mis metodos abstractos
  public abstract void acelerar(float cantidad);

  public abstract void frenar(float cantidad);

  public abstract String toString();

  public void setVelocidad(float velocidad){
    this.velocidad = velocidad;
  }
}

File: LOO 2°Parcial/Simulador de Vehiculos CSharp/Simulador de Vehiculos CSharp/Vehiculo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Simulador_de_Vehiculos_CSharp
{
    abstract class Vehiculo
    {
        protected String matricula;
        protected float velocidad;
        protected float cantidad;

        public Vehiculo(String matricula, float velocidad)
        {
            this.matricula = matricula;
            this.velocidad = velocidad;
        }
        public float Velocidad
        {
            set { this.velocidad = value; }
            get { return this.velocidad; }
        }

        public abstract void acelerar(float cantidad);

        public abstract void frenar(float cantidad);

        public abstract String toString();

        public void setVelocidad(float velocidad)
        {
            this.velocidad = velocidad;
        }
    }
}
